src/main/java/edu/kis/powp/jobs2d/usage/TankObserver.java

```diff
@@ -0,0 +1,7 @@
+package edu.kis.powp.jobs2d.usage;
+
+import edu.kis.powp.jobs2d.enums.DeviceManagementWarnings;
+
+public interface TankObserver {
+    void update(DeviceManagementWarnings warning);
```

Comment:
Passing a Tank instance instead of 'warning' would be a more configurable option.